### ISSUE TITLE
GetTxStatus script returns NullReferenceException

### ISSUE THREAD

**Issue by juans-chainsafe** [ChainSafe/web3.unity#521]
**Describe the bug**
After you sign the transaction in Metamask for the tx status, you get an error in the console on `var txReceipt = await tx.Wait();`
The transaction in Metamask is completed, so probably something in our code.

**To Reproduce**
1. Add an empty game object
2. Drag and drop **GetTxStatus** to the game object
3. Click play with the LoginScene first
4. Do the login with Metamask
5. Sign the transaction
6. See error

**Expected behavior**
Transaction receipt appears in the console

**Additional context**
```
NullReferenceException: Object reference not set to an instance of an object
GetTxStatus.Start () (at Assets/Scripts/Samples/Wallet/GetTxStatus.cs:15)
System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__7_0 (System.Object state) (at <b6c5d0f08ddf477186f780f49bb14dee>:0)
UnityEngine.UnitySynchronizationContext+WorkRequest.Invoke () (at /Users/bokken/build/output/unity/unity/Runtime/Export/Scripting/UnitySynchronizationContext.cs:153)
UnityEngine.UnitySynchronizationContext.Exec () (at /Users/bokken/build/output/unity/unity/Runtime/Export/Scripting/UnitySynchronizationContext.cs:83)
UnityEngine.UnitySynchronizationContext.ExecuteTasks () (at /Users/bokken/build/output/unity/unity/Runtime/Export/Scripting/UnitySynchronizationContext.cs:107)

```
<img width="1050" alt="Screenshot 2023-07-10 at 13 48 15" src="https://github.com/ChainSafe/web3.unity/assets/104365141/82d221e2-2fa1-4806-b6fa-dec02b4d82e3">

**Comment by Arshia001** [comment]
Fixed in #486

**Comment by juans-chainsafe** [comment]
Tested:

- GetTxStatus
- SendArray
- ContractCall
- Transfer20

All working correctly now.
Just one comment: the log prints System.object[] so is not a great log, we should improve that in the future
